.jshintrc

```diff
@@ -1,4 +1,5 @@
 {
+  "esversion": 6,
 	"bitwise" : true,
```

Comment:
come on, you can do better than that 😆

Reply:
You are welcome to submit a PR that addresses all of our current linting violations. I would rather switch to `eslint` anyway, move the project to another repo where I have admin rights, and enabled Github CI that automatically runs linting as well as the tests.

Reply:
Oh, just thought it would make sense to stay consistent, and not to change the indentation on a per line basis. I don't really care about the linting otherwise :)